<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#3a025b</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTextBrowser" name="textEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>341</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#FFF1DB</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>481</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#FFF1DB
</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Coffee_widgets">
      <attribute name="title">
       <string>Coffee</string>
      </attribute>
      <widget class="QPushButton" name="caramel_bt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Caramel 130</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chocolate_bt">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Chocolate 100</string>
       </property>
      </widget>
      <widget class="QPushButton" name="vanilla_bt">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Vanilla 120</string>
       </property>
      </widget>
      <widget class="QPushButton" name="melon_bt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Melon Milk 90</string>
       </property>
      </widget>
      <widget class="QPushButton" name="avocado_bt">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Avocado 100</string>
       </property>
      </widget>
      <widget class="QPushButton" name="straberry_bt">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Straberry 90</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>210</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Frappe</string>
      </attribute>
      <widget class="QPushButton" name="hazelnut_bt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Hazelnut</string>
       </property>
      </widget>
      <widget class="QPushButton" name="okinawa_bt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Okinawa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="java_bt">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Java 140</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>210</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#000000</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../.designer/backup/Chocolate.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="red_velvet_bt">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>330</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Red Velvet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mango_bt">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mango</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mocha_bt">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Mocha</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="check_out_bt">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#FFF1DB
</string>
     </property>
     <property name="text">
      <string>Check out</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>410</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#FFF1DB</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTreeView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>451</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#FFF1DB</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
